fingerprint_report_templates/fingerprint_report/templates/macros.html

```diff
@@ -24,19 +24,19 @@
     {% endif %}
 {%- endmacro %}
 
-
+{# draw a generic pandas table#}
 {% macro pandas_table(df, caption, column_labels={}) -%}
```

Comment:
add the verb to the method: `render_pandas_table` to be more explicit about what the macro does